pkg/kind/README.md

```diff
@@ -0,0 +1 @@
+# COPIED FROM KIND INTERNAL PACKAGE AND MODIFIED TO FIT THE NEEDS OF A CAPK PROVIDER
```

Comment:
Everything inside pkg/kind is copy pasted from kind/pkg/internal. It is somewhat modified, but only to integrate with code outside of the kind library.

This can mostly be skipped.